app/src/main/resources/application.properties

```diff
@@ -27,6 +27,7 @@ quarkus.log.console.format=%d{YYYY-MM-DD HH:mm:ss} %p <%X{tenantId}> [%C] (%t) %
 
 
 # Log
+%prod.quarkus.log.min-level=TRACE
```

Comment:
What is min-level?

Reply:
something from quarkus I faced when trying to see the logs for the new auth stuff. https://quarkus.io/guides/logging#runtime-configuration